plugins/adobe-analytics-compatible-version/index.ts

```diff
@@ -104,9 +105,9 @@ import { ValidationPluginResult } from 'types/validationPlugin';
           "No version info could be found. Either the Assurance SDK isn't registered or the SDK did not pass in cached events upon activating.",
         result: 'unknown'
       }
-    : !compatible
+    : !compatible && analyticsVersion
     ? {
-        message: `Assurance SDK Version ${assuranceVersion} and Adobe Analytics Extension version ${analyticsVersion} are not compatible!`,
+        message: `Assurance SDK Version ${assuranceVersion} and Adobe Analytics extension version ${analyticsVersion} are not compatible!`,
```

Comment:
can we also indicate what is the resolution for this error? e.g.` ... are not compatible. Check the link for latest versions available and upgrade to resolve this error.`